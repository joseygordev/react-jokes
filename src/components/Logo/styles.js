export default {
  root: {
    width: '80%',
    margin: '5%',
    maxWidth: 400
  }
};