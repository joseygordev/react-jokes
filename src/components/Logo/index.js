import React from 'react';

// Styles
import styles from './styles';

const Logo = () => (
  <img style={styles.root} src="https://api.chucknorris.io/img/chucknorris_logo_coloured_small@2x.png" alt="Chuck Norris" />
);

export default Logo;
