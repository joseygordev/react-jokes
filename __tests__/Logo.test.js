import React from 'react';
import renderer from 'react-test-renderer';
import Logo from '../src/components/Logo';

describe('Logo', () => {
  it('Should return the Logo component', () => {
    const tree = renderer
      .create(<Logo />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
