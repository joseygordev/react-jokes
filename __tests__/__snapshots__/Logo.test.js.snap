// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo Should return the Logo component 1`] = `
<img
  alt="Chuck Norris"
  src="https://api.chucknorris.io/img/chucknorris_logo_coloured_small@2x.png"
  style={
    Object {
      "margin": "5%",
      "maxWidth": 400,
      "width": "80%",
    }
  }
/>
`;
